src/work-allocation/containers/available-tasks/available-tasks.component.spec.ts

```diff
@@ -101,7 +101,8 @@ describe('AvailableTasksComponent', () => {
         { provide: AllocateRoleService, useValue: mockRoleService },
         { provide: Store, useValue: storeMock },
         { provide: RpxTranslationService, useFactory: rpxTranslationServiceStub }
-      ]
+      ],
+      schemas: [NO_ERRORS_SCHEMA]
```

Comment:
This is usually a hack. Before release we should figure out why this is needed, and remove it unless there is some very special circumstance that requires it.